packages/api/src/auth/github/github.service.spec.ts

```diff
@@ -50,14 +50,13 @@ describe('Github controller', () => {
     it("Creates and returns a new user when `githubId` doesn't match any record in db", async () => {
       const newUser = createUser(222);
 
-      usersRepository.getByGithubId = jest.fn().mockResolvedValue(null);
+      usersRepository.findByGithubId = jest.fn().mockResolvedValue(null);
       usersRepository.create = jest.fn().mockResolvedValue(newUser);
 
-      const githubUser = createObjectMock<UserFromGithub>({ githubId: newUser.githubId });
-
+      const githubUser = createObjectMock<NotRegisteredUser>({ githubId: newUser.githubId });
       const result = await service.authorizeUser(githubUser);
 
-      expect(usersRepository.getByGithubId).toHaveBeenCalledWith(newUser.githubId);
+      expect(usersRepository.findByGithubId).toHaveBeenCalledWith(newUser.githubId);
```

Comment:
It's testing of implementation details, not business logic. Tests should defend you against regression while refactor. There refactor will break tests.